shared/chat/conversation/messages/unfurl/prompt/index.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../../../../common-adapters/index'
+import * as Styles from '../../../../../styles'
+
+export type Props = {
+  domain: string,
+  onAlways: () => void,
+  onAccept: () => void,
+  onNotnow: () => void,
+  onNever: () => void,
+}
+
+const promptIcon = Styles.isMobile
+  ? 'icon-fancy-unfurl-preview-mobile-128-128'
+  : 'icon-fancy-unfurl-preview-desktop-96-96'
+
+class UnfurlPrompt extends React.PureComponent<Props> {
+  render() {
+    return (
+      <Kb.Box2 direction="horizontal" style={styles.container}>
+        <Kb.Icon type={promptIcon} style={Kb.iconCastPlatformStyles(styles.icon)} />
+        <Kb.Box2 direction="vertical" style={styles.choiceContainer}>
+          <Kb.Text type="BodySemibold">Would you like to post a preview?</Kb.Text>
+          <Kb.Text type="Body">Your Keybase app will visit the link and post a preview of it.</Kb.Text>
+          <Kb.Text onClick={this.props.onAlways} type="BodyPrimaryLink">
+            Always for any site.
+          </Kb.Text>
+          <Kb.Text onClick={this.props.onAccept} type="BodyPrimaryLink">
+            Always, for {this.props.domain}.
+          </Kb.Text>
+          <Kb.Text onClick={this.props.onNotnow} type="BodyPrimaryLink">
+            Not now.
+          </Kb.Text>
+          <Kb.Text onClick={this.props.onNever} type="BodyPrimaryLink">
+            Never, for any site.
+          </Kb.Text>
+        </Kb.Box2>
+        <Kb.Box2 direction="horizontal" style={styles.closeContainer}>
+          <Kb.Icon type="iconfont-close" onClick={this.props.onNotnow} fontSize={16} />
+        </Kb.Box2>
+      </Kb.Box2>
+    )
+  }
+}
+
+const styles = Styles.styleSheetCreate({
+  container: {
+    ...Styles.globalStyles.flexBoxRow,
+    backgroundColor: Styles.globalColors.blue5,
+    borderRadius: Styles.borderRadius,
+    paddingTop: 8,
+    paddingBottom: 8,
+  },
+  choiceContainer: {
+    width: 370,
```

Comment:
is the intention of this PR that this works on the phone also? If so this is likely incorrect. We always build against an iphone5s sim to ensure things look ok on small form factor phones. Although this likely passes flow, we don't strictly type a bunch of the css. it just types as string, so things like `marginLeft: 'auto'` below work (as we want to allow `marginLeft: 120.3px` or whatever, but auto isn't a thing in RN

Reply:
There isn't a design on the phone yet, so I didn't really look at it there. I was just going to have this off for phone in the short term. I could drop this in for platform specific right?

Reply:
`platformStyles` or whatever

Reply:
the preferred solution is to use 
```
Styles.platformStyles({
  common: {},
  isElectron: {},
  isMobile: {},
})
```

Reply:
I'll change it to that then.